<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtArteryTechLinearSolverSetupUI</class>
 <widget class="QDialog" name="QtArteryTechLinearSolverSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtArteryTechLinearSolverSetupUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>线性求解器类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LinearSolverTypeLabel">
             <property name="text">
              <string>Linear Solver Type:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="LinearSolverTypeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>预处理方向</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="PreconditioningSideLabel">
             <property name="text">
              <string>Preconditioning Side:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="PreconditioningSideLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>重启数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="RestartNumberLabel">
             <property name="text">
              <string>Restart Number:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="RestartNumberLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>最大迭代次数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="MaxIterationNumberLabel">
             <property name="text">
              <string>Max Iteration Number:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="MaxIterationNumberLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>线性相对收敛误差</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="RelativeTolleranceLabel">
             <property name="text">
              <string>Relative Tollerance:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="RelativeTolleranceLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>线性绝对收敛误差</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="AbsoluteTolleranceLabel">
             <property name="text">
              <string>Absolute Tollerance:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="AbsoluteTolleranceLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>第一步线性收敛相对误差</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Different1stStepTolLabel">
             <property name="text">
              <string>Different 1st Step Tol:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="Different1stStepTolLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>第一步线性收敛相对误差比率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="RatioOfThe1stStepTolLabel">
             <property name="text">
              <string>Ratio of the 1st Step Tol:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="RatioOfThe1stStepTolLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="OkPushButton">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
